### PR TITLE
Hotfix/yard

### PR DESCRIPTION
**Pull request by ctlong**
- Fixes an issue with yard that was preventing it from compiling
- Init Travis CI rspec and yard testing in build

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -5,9 +5,13 @@
 RSpec::Expectations.configuration.warn_about_potential_false_positives = false
 
 # set environment variable from .env
-lines = File.read '.env'
-lines = lines.split('\n')
-lines.each do |line|
-  line = line.split(' = ')
-  ENV[line[0]] = line[1].gsub!('"', '')
+begin
```
Comment (wingleungchoi) [inline spec/spec_helper.rb]: maybe we shall use `if` rather than rescue
as if is 10 times faster than rescue

`if File.exist?(.env)`

**Comment by wingleungchoi** [comment]
@ctlong using travis 👍 , agile + bugs-less
just one comment

File: spec/spec_helper.rb (hunk 1 above)
Comment (ctlong) [inline spec/spec_helper.rb]: Good idea!

**Comment by wingleungchoi** [comment]
@ctlong 👍

**Pull request merged**